mono/metadata/metadata.c

```diff
@@ -1842,10 +1842,11 @@ mono_metadata_parse_type_internal (MonoImage *m, MonoGenericContainer *container
 }
 
 
-MonoType*
+MONO_API MonoType*
```

Comment:
If the declaration in the `.h` file is marked as `MONO_API` then there is no need to do it in the `.c` file too.